pkgs/development/interpreters/tcl/8.5.nix

```diff
@@ -12,5 +12,9 @@ callPackage ./generic.nix (
       url = "mirror://sourceforge/tcl/tcl${version}-src.tar.gz";
       sha256 = "066vlr9k5f44w9gl9382hlxnryq00d5p6c7w5vq1fgc7v9b49w6k";
     };
+
+    extraPatch = ''
+      substituteInPlace 'generic/tclInt.h' --replace-fail 'typedef int ptrdiff_t;' ""
+    '';
```

Comment:
I don't understand, isn't #369404 fixing the same issue differently, and already merged?

Reply:
Oh right. I must have used an old checkout of master when starting this PR.

Reply:
Actually, this is a fix for tk-8_5. The fix is just done in the headers of tcl 8.5. I must reword the commit message.

Or maybe there is a better fix for tk 8.5